src/main/java/com/example/gradecalculator/model/UserSubjectsTO.java

```diff
@@ -0,0 +1,20 @@
+package com.example.gradecalculator.model;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.List;
+
+@Getter
+@Setter
+public class UserSubjectsTO {
+
+    private int userId;
+    private List<SubjectTO> selectedSubjects;
+
+    public UserSubjectsTO(int userId, List<SubjectTO> selectedSubjects, boolean termsAccepted) {
```

Comment:
what is this constructor needed for?

Reply:
I don't remember but UserSubjectsTO didn't habe any usage and i've deleted it